--- "C:\\Users\\warren\\Documents\\Arduino\\libraries\\ESP32-BLE-Keyboard\\~BleKeyboard.h"	2021-02-23 19:48:35.515474800 +0800
+++ "C:\\Users\\warren\\Documents\\Arduino\\libraries\\ESP32-BLE-Keyboard\\BleKeyboard.h"	2021-02-23 19:42:03.267057500 +0800
@@ -96,6 +96,7 @@
   BLECharacteristic* inputMediaKeys;
   KeyReport _keyReport;
   MediaKeyReport _mediaKeyReport;
+  BLEServer *pServer;
   static void taskServer(void* pvParameter);
 public:
   BleKeyboard(std::string deviceName = "ESP32 BLE Keyboard", std::string deviceManufacturer = "Espressif", uint8_t batteryLevel = 100);
