--- "C:\\Users\\warren\\Documents\\Arduino\\libraries\\ESP32-BLE-Keyboard\\~BleKeyboard.cpp"	2021-02-23 19:48:50.054823200 +0800
+++ "C:\\Users\\warren\\Documents\\Arduino\\libraries\\ESP32-BLE-Keyboard\\BleKeyboard.cpp"	2021-02-23 19:45:58.285475900 +0800
@@ -104,6 +104,7 @@
 
 void BleKeyboard::end(void)
 {
+	this->pServer->disconnect(this->pServer->getConnId());
 }
 
 bool BleKeyboard::isConnected(void) {
@@ -120,6 +121,7 @@
   BleKeyboard* bleKeyboardInstance = (BleKeyboard *) pvParameter; //static_cast<BleKeyboard *>(pvParameter);
   BLEDevice::init(bleKeyboardInstance->deviceName);
   BLEServer *pServer = BLEDevice::createServer();
+  bleKeyboardInstance->pServer = pServer;
   pServer->setCallbacks(bleKeyboardInstance->connectionStatus);
 
   bleKeyboardInstance->hid = new BLEHIDDevice(pServer);
